src/settings/handlers/DeviceSettingsHandler.ts

```diff
@@ -106,6 +114,18 @@ export default class DeviceSettingsHandler extends SettingsHandler {
             return Promise.resolve();
         }
 
+        // Special case for old useIRCLayout setting
```

Comment:
Since this is a device only level, is there any need to _write_ the old value? It seems like it would only be needed if you are going back in time to old versions, which seems unlikely. (If it were account level and synced to other devices, we would need it, but that doesn't seem to be the case here.)

Reply:
This if statement is here to _delete_ `useIRCLayout`.  
```ts
if (settings["useIRCLayout"]) return Layout.IRC;
``` 
Otherwise, this would always be true.

Reply:
Ah okay, fair enough. I suppose alternatively the read path could always check the modern value first and stop there if it exists, but anyway, this seems okay too.